### DISCUSSION TITLE
Will my recuring Job perform missed-slots ?

### DISCUSSION THREAD

**Discussion by benallard** [jobrunr/jobrunr#315] [Q&A]
Say I have a daily Job, everyday at 7am.

For some obscure reasons, my server is down from 5 to 9 on that day. Will JobRunr start up the "missed" slot at 9 (upon fresh start), or will the next time be tomorrow at 7 (if the server is up)?

**Comment by rdehuyss** [comment]
The day after.

**Reply by benallard** [reply]
Do you have any hint if / how I could achieve what I'm looking for, possibly by using as much from jobrunr as possible. Would you be interested in such a feature?

**Comment by rdehuyss** [comment]
I think it will be really hard to implement it well... If you have ideas on how to do it, (using pseudo-code) feel free to elaborate here.

**Reply by benallard** [reply]
Maybe using a combination of duration-based configuration and cron-based configuration, whichever comes first. This would requires to store the time of last successful run (or time of last queing?). yeah, if it's been queued, the scheduler is out, his job is done. No matter if it ran successfully or not. What do you think ?

**Reply by rdehuyss** [reply]
This is not completely clear for me what you mean.

Why not keep track of it in your own DB?
